docs/.vitepress/config.js

```diff
@@ -72,6 +72,10 @@ export default {
       prev: false,
       next: true,
     },
+    editLink: {
```

Comment:
Edit link not working. Should follow [docs format](https://vitepress.dev/reference/default-theme-config#editlink)
```js
https://github.com/Evavic44/adocs/edit/main/docs/:path
```